hack/boilerplate/boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (C) YEAR Red Hat, Inc.
+Copyright YEAR The Kubernetes Authors All rights reserved.
```

Comment:
@hferentschik Never thought of putting the copyrightholder as a token as I thought it wont be elegant to put the whole paragraph as a token. But lets try that.